runregistry-rest/database.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+
+from api import db
+
+
+__all__ = ["RunRegistryConfig", "RunRegistryMeta"]
+
+
+class RunRegistryConfig(db.Model):
+    rn = db.Column(
```

Comment:
Should this have a foreign key relationship to `RunRegistryMeta`?

Reply:
Should there also be a foreignkey relationship with RunNumber database too?

Reply:
I'm not sure how to define foreign keys across applications, but I'd love for there to be some sort of sane relationship between them.

Reply:
Would it make sense to merge these two API's together?  Without `RunNumber` defined I'm not sure that foreign key would work as expected...

Reply:
I may consider it, if I don't manage to figure this out

Reply:
Looks like it didn't work as expected, should i actually merge the API's? Should I leave the foreignkeys to RunNumber seperate? Or should I present at the next CCM meeting and wait until we discuss? (I realise this may be a while before we have one of those meetings)

Reply:
I'm inclined towards merging, but we should probably discuss at CCM.